plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java

```diff
@@ -3065,56 +3063,6 @@ public void testEmptyTableCreation()
         }
     }
 
-    @Test
-    public void testCreateEmptyTableShouldNotCreateStagingDirectory()
```

Comment:
Does this test need to be deleted ? It looks possible to make it work without session property overrides as well

Reply:
The case the test is verifying just didn't seem that important to me but if you think it is worth having I can take a stab at rewriting it

Reply:
It was added in https://github.com/trinodb/trino/pull/14830 to fix something, so looks important enough to retain

Reply:
Can do. That wound up being easier than I thought, switched it to use a random user to ensure no directory overlap instead of the session property.